<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusViewRubbishBoat</class>
 <widget class="QWidget" name="StatusViewRubbishBoat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1013</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWebEngineView" name="mapWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>9</y>
     <width>960</width>
     <height>613</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="videoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>931</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="VlcWidgetVideo" name="videoPlayWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="videoControlWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="videoAddEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(115, 210, 22);</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(239, 41, 41);</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="statusWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>630</y>
     <width>1891</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 16, 48);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="SpeedWidget2" name="widget" native="true"/>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="AttitudeWidget" name="widget_2" native="true"/>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="AltitudeWidget" name="widget_3" native="true">
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>16</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>290</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>101</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string>参数信息：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>151</width>
         <height>112</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>转向角：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="angle_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_x：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="ax_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_y：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="ay_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_z：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="az_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>221</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>任务设置</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>参数调整</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>执行任务</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>数据记录</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>紧急恢复</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>断开连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttitudeWidget</class>
   <extends>QWidget</extends>
   <header>attitudewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AltitudeWidget</class>
   <extends>QWidget</extends>
   <header>altitudewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpeedWidget2</class>
   <extends>QWidget</extends>
   <header>speedwidget2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
